db-scheme/mapybezbarier.rdf

```diff
@@ -966,6 +966,13 @@
     ];
 .
 
+<#EntityWcCabinAccess>
+    rr:logicalTable [ rr:tableName "mappable_entity__wc_cabin_access" ];
```

Comment:
Typo `mappable_entity__wc_cabin_access` => `mappable_entity_wc_cabin_access`

Jinak je to ok.